examples/basic_usage/main.tf

```diff
@@ -50,89 +38,18 @@ resource "aws_route" "route_external_traffic_through_internet_gateway" {
 }
 
-
 #-------------------------------------------------------------------------------
-# Create a private Route53 zone.
+# Configure the server module.
 #-------------------------------------------------------------------------------
-resource "aws_route53_zone" "private_zone" {
-  name = "cyber.dhs.gov"
-
-  vpc {
-    vpc_id = aws_vpc.the_vpc.id
-  }
-}
-
-#-------------------------------------------------------------------------------
-# Create a private Route53 reverse zone.
-#-------------------------------------------------------------------------------
-resource "aws_route53_zone" "master_private_reverse_zone" {
-  name = "48.99.10.in-addr.arpa"
-
-  vpc {
-    vpc_id = aws_vpc.the_vpc.id
-  }
-}
-
-#-------------------------------------------------------------------------------
-# Create a data resource for the existing public Route53 zone.
-#-------------------------------------------------------------------------------
-data "aws_route53_zone" "public_zone" {
-  provider = aws.public_dns
-
-  name = "cyber.dhs.gov."
-}
-
-#-------------------------------------------------------------------------------
-# Create a role that allows the master to read its certs from S3.
-#-------------------------------------------------------------------------------
-data "aws_caller_identity" "shared_services" {
-}
-
-module "certreadrole" {
-  source = "github.com/cisagov/cert-read-role-tf-module"
-
-  providers = {
-    aws = aws.cert_read_role
-  }
-
-  account_ids = [
-    data.aws_caller_identity.shared_services.account_id,
-  ]
-  cert_bucket_name = "cisa-cool-certificates"
-  hostname         = "ipa.cal23.cyber.dhs.gov"
-}
-
-#-------------------------------------------------------------------------------
-# Configure the master module.
-#-------------------------------------------------------------------------------
-module "ipa_master" {
+module "ipa" {
   source = "../../"
 
-  providers = {
-    aws            = aws
-    aws.public_dns = aws.public_dns
-  }
-
-  admin_pw                    = "thepassword"
-  ami_owner_account_id        = "207871073513" # The COOL Images account
-  associate_public_ip_address = true
-  cert_bucket_name            = "cisa-cool-certificates"
-  cert_pw                     = "lemmy"
-  cert_read_role_arn          = module.certreadrole.role.arn
-  directory_service_pw        = "thepassword"
-  domain                      = "cal23.cyber.dhs.gov"
-  hostname                    = "ipa.cal23.cyber.dhs.gov"
-  private_reverse_zone_id     = aws_route53_zone.master_private_reverse_zone.zone_id
-  private_zone_id             = aws_route53_zone.private_zone.zone_id
-  public_zone_id              = data.aws_route53_zone.public_zone.zone_id
-  realm                       = "CAL23.CYBER.DHS.GOV"
-  subnet_id                   = aws_subnet.master_subnet.id
+  ami_owner_account_id = "207871073513" # The COOL Images account
+  domain               = "cal23.cyber.dhs.gov"
```

Comment:
What is this `cal23` jazz?

Reply:
That has been there for a long time.  It's just a rando example domain that @jsf9k picked to avoid confusion (ironically) with any COOL domains.